ksql-engine/src/test/java/io/confluent/ksql/planner/plan/DataSourceNodeTest.java

```diff
@@ -354,25 +355,9 @@ public void shouldBuildTableByConvertingFromStream() {
     final SchemaKStream returned = node.buildStream(ksqlStreamBuilder);
 
     // Then:
-    verify(stream).toTable(any(), any(), any());
     assertThat(returned, is(table));
   }
 
-  @Test
-  public void shouldBuildTableWithCorrectContext() {
```

Comment:
no longer possible to test this here. Instead test added to `SourceBuilderTest` called `shouldBuildTableWithCorrectStoreName`.